Vagrantfile

```diff
@@ -95,6 +95,12 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
       app.vm.synced_folder "src/rf/apps", "/opt/app/apps"
     else
       app.vm.synced_folder "src/rf", "/opt/app/"
+      app.vm.synced_folder "#{ENV['HOME']}/.aws", "/var/lib/rf/.aws/",
+        owner: "rf",
```

Comment:
May want to try mounting the credential directory into `/aws` and then use `BOTO_CONFIG` to signal the app to use `/aws` vs. `~/.aws`.

What was the error before?